src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs

```diff
@@ -0,0 +1,239 @@
+            SetTestParameters(_parameters);
+        }
+
+        [TearDown]
+        public Task DisposeAsync()
+        {
+            _parameters.Dispose();
+            return Task.CompletedTask;
+        }
+
+        [Test]
+        public async Task Subscriptions_EndToEnd()
+        {
+            TestContext.WriteLine("Subscription management tests...");
+
+            string repo1Uri = GetRepoUrl(repo1Name);
+            string repo2Uri = GetRepoUrl(repo2Name);
+            string repo3Uri = GetRepoUrl(repo3Name);
+            string repo1AzDoUri = GetAzDoRepoUrl(repo1Name);
+            string targetBranch = "master";
+
+            TestContext.WriteLine($"Creating channels {channel1Name} and {channel2Name}");
+            await using (AsyncDisposableValue<string> channel1 = await CreateTestChannelAsync(channel1Name).ConfigureAwait(false))
+            {
+                await using (AsyncDisposableValue<string> channel2 = await CreateTestChannelAsync(channel2Name).ConfigureAwait(false))
+                {
+
+                    TestContext.WriteLine("Testing various command line parameters of add-subscription");
+                    await using AsyncDisposableValue<string> subscription1Id = await CreateSubscriptionAsync(
+                        channel1Name, repo1Name, repo2Name, targetBranch, "everyWeek", "maestro-auth-test");
+
+                    Subscription expectedSubscription1 = new Subscription(Guid.Parse(subscription1Id.Value), true, repo1Uri, repo2Uri, targetBranch);
+                    expectedSubscription1.Channel = new Channel(42, channel1Name, "test");
+                    expectedSubscription1.Policy = new SubscriptionPolicy(false, UpdateFrequency.EveryWeek);
+                    expectedSubscription1.Policy.MergePolicies = ImmutableList<MergePolicy>.Empty;
+
+                    string expectedSubscription1Info = SubscriptionStringBuilder.GetSubscriptionString(expectedSubscription1);
+
+                    await ValidateSubscriptionInfo(subscription1Id.Value, expectedSubscription1Info);
+
+                    await using AsyncDisposableValue<string> subscription2Id = await CreateSubscriptionAsync(
+                        channel1Name, repo1Name, repo1Name, targetBranch, "none", "maestro-auth-test",
+                        new List<string>
+                        { "--all-checks-passed", "--no-extra-commits", "--no-requested-changes", "--ignore-checks", "WIP,license/cla" }, targetIsAzDo: true);
+
+                    Subscription expectedSubscription2 = new Subscription(Guid.Parse(subscription2Id.Value), true, repo1Uri, repo1AzDoUri, targetBranch);
+                    expectedSubscription2.Channel = new Channel(42, channel1Name, "test");
+                    expectedSubscription2.Policy = new SubscriptionPolicy(false, UpdateFrequency.None);
+
+                    string expectedPoliciesSub2 = "  - Merge Policies:\r\n    NoExtraCommits\r\n    AllChecksSuccessful\r\n      ignoreChecks = \r\n" +
```

Comment:
Use a verbatim string, instead of a bunch of normal string with new lines embedded an contatenated together.